INSTALLATION_TROUBLESHOOTING.md

```diff
@@ -35,6 +35,19 @@
     2. When you run ElasticSearch, it seems that the process takes a very long time. 
 	To proceed the process, just let ElasticSearch to run
     on its own terminal, and proceed the building process by making another window of terminal
+	
+	3. If you run into an error on Docker Toolbox with Elastic Search stating an error message along the lines of:
+		"Waiting for elasticsearch:9200  .elasticsearch: forward host lookup failed: Unknown host"
+	   The cause could be the docker-machine's memory limits. you can inspect this with the command: docker-machine inspect machine-name
+	   
+	   To solve this problem, please try recreating the default docker machine by:
+	   		i. Remove default docker-machine with the command: 
+				docker-machine rm default
```

Comment:
Here and for subsequent snippets of code, please use backticks (this character: ``, which is not a single quote ) around the snippet like quotes so that it is formatted as code `like this`.

You can also look for the "<>" button in the text editor to add it automatically

Reply:
You can also do multiline snippets  with three backticks on their own line before and after your code
```
Like this
multiline
no problem
```